www/analytics/last-score-by-activity-by-user/last-score-by-activity-by-user-cut-score1.php

```diff
@@ -0,0 +1,169 @@
+    ],
+]);
+?>
+
+<!--visualization checkboxes-->
+<style>
+    /* Styles for the checkbox when checked */
+     input[type="checkbox"].highlight-band-fail1-switch:checked {
+        background-color:#34C759;
+        width: 38px;
+        height:23px;
+        border-color: #34C759;
+    }
+    /* Styles for the checkbox when not checked */
+     input[type="checkbox"].highlight-band-fail1-switch1:not(:checked) {
+        width: 38px;
+        height:23px;
+    }
+    .lrn-cut-score-switch {
+        margin-bottom: 15px;
+    }
+    .switch-input {
+        display: none;
+    }
+    .switch-label {
+        display: block;
+        width: 40px;
+        height: 24px;
+        background-color: gray;
+        border-radius: 15px;
+        position: relative;
+        cursor: pointer;
+        transition: background-color 0.3s ease;
+    }
+    .switch-label:before {
+        content: '';
+        position: absolute;
+        top: 2px;
+        left: 2px;
+        width: 20px;
+        height: 20px;
+        background-color: white;
+        border-radius: 50%;
+        transition: left 0.3s ease;
+    }
+    .switch-input:checked + .switch-label {
+        background-color: #34C759;
+    }
+    .switch-input:checked + .switch-label:before {
+        left: calc(100% - 22px);
+    }
+    .switch-text {
+        margin-left: 50px;
+        width: 100px;
+        display: inline-block
+    }
+    .learnosity-report.lastscore-by-activity-by-user .lrn-report-table > tbody > tr > td.lrn-report-clickable {
+        position: relative;
+        vertical-align: middle;
+    }
+
+    .learnosity-report.lastscore-by-activity-by-user .lrn-report-table > tbody > tr > td.lrn-report-clickable:hover::before {
+        content: '';
+        position: absolute;
+        top: 1px;
+        left: 1px;
+        right: 1px;
+        bottom: 1px;
+        box-shadow: 0 0 0 1px #696969;
+    }
+
+    .learnosity-report.lastscore-by-activity-by-user .lrn-report-table > tbody > tr > td.lrn-report-clickable:focus {
+        border: 2px solid #333333;
+    }
+</style>
+<!--visualization performance band fail styles-->
+<style id="band-fail-styles-last-score-by-activity">
+    .learnosity-report.lastscore-by-activity-by-user .lrn-report-table > tbody > tr > td[data-custom_performance="fail2"] {
+        background-color: #FBE3E3;
+        font-weight: 700;
+    }
+
+    .learnosity-report.lastscore-by-activity-by-user .lrn-report-table > tbody > tr > td[data-custom_performance="fail2"]::before {
+        content: '';
+        position: absolute;
+        top: 2px;
+        left: 2px;
+        right: 2px;
+        bottom: 2px;
+        box-shadow: 0 0 0 2px #DD002F;
+    }
+</style>
+
+<div class="lsbabu-report"  id="lsbabu-report-cutscore1" style="display: none">
+    <div class="form-check form-switch lrn-cut-score-switch" >
+
+    <input type="checkbox" id="highlight-band-fail1" class="switch-input highlight-band-fail1-switch1" checked >
```

Comment:
I think if you put class `highlight-band-fail-switch1` looks better than `highlight-band-fail1-switch1`, but if you can use a better variable name that will be good.